kardinal-manager/kardinal-manager/cluster_manager/cluster_manager.go

```diff
@@ -16,11 +16,12 @@ import (
 )
 
 const (
-	listOptionsTimeoutSeconds       int64 = 10
-	fieldManager                          = "kardinal-manager"
-	deleteOptionsGracePeriodSeconds int64 = 0
-	istioLabel                            = "istio-injection"
-	enabledIstioValue                     = "enabled"
+	listOptionsTimeoutSeconds         int64 = 10
+	fieldManager                            = "kardinal-manager"
+	deleteOptionsGracePeriodSeconds   int64 = 0
+	istioLabel                              = "istio-injection"
+	enabledIstioValue                       = "enabled"
+	telepresenceRestartedAtAnnotation       = "telepresence.getambassador.io/restartedAt"
```

Comment:
@lostbean I'm not using this annotation to check if the operator is in the cluster I'm using this to detect if this annotation is in the current `deployment` and if it is there and keep it there, otherwise the apply mechanism will create a new revision for the deployment without this label and this will trigger the pod remotion.